README.md

```diff
@@ -8,9 +8,19 @@ An app to handle subscription management on cloud.redhat.com, built using the [S
 
 1. The first time you clone the repo, run `npm install`
 
-2. `npm run start` or `npm run start:beta`, depending on whether you want to run in stable or beta mode.
+2. Add IPs to /etc/hosts by running `vi /etc/hosts`, then add the following:
```

Comment:
This command has to be run as root.

Also worth noting that this is only proven to work on Linux machines. I think on a Mac the command is the same but its very different on Windows. I don't think we need to add how to do it on every operating system but should note that it may be different if the user isn't on Linux.